dev/build_docker_images.sh

```diff
@@ -4,35 +4,49 @@
 #   os
 #   docker_username
 #   docker_password
+#   GITHUB_REF
 
 python_version="3.8"
-tags=""
 version="$(cat version)"
 
+set -ue
+
+git_tag="${GITHUB_REF#refs/tags/v*}"
```

Comment:
Not sure if that's something desirable to have in here; what do you think @gdubicki

Reply:
I think that we should be consistent from where do we get the version for both Docker images and the PyPI releases.